packages/fast-components-react-base/src/listbox/listbox.props.ts

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import {
+    ListboxClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface ListboxManagedClasses extends ManagedClasses<ListboxClassNameContract> {}
+export interface ListboxUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+export interface ListboxHandledProps extends ListboxManagedClasses {
+    /**
+     * The listbox children
+     */
+    children?: React.ReactNode;
+
+    /**
+     * The name of the prop on child components that is used as the data for type ahead focus
+     */
+    typeAheadPropertyKey?: string;
+
+    /**
+     * Whether this listbox supports multi-selection (default is 'false')
+     */
+    multiselectible?: boolean;
```

Comment:
selectable*

I think though you could just remove this all-together as unhandledProps will allow this to fall through

Reply:
Eh - I'm not sure we want to leave this to unhandled. There is what seems to be significantly different keyboard interaction/behavior with multi-select. If we support it (I think we should at base), I think we should handle it; otherwise how do they manage their focus for just that scenario?

Reply:
Fixed the typo.  But I agree with Chris.  I realized I need to refactor to move selection logic that I hoped could live in the parent "Select" control into the listbox itself so this prop will do more than just flag whether we need to aria label the listbox as multselect.

Reply:
Multi-selectable does much more in "listbox manages selection" refactor.